controlplane/src/core/services/Keycloak.ts

```diff
@@ -70,6 +76,55 @@ export default class Keycloak {
     return createUserResp.id;
   }
 
+  public async updateKeycloakUser({
+    realm,
+    id,
+    password,
+    firstName,
+    lastName,
+    groups,
+    enabled,
+  }: {
+    id: string;
+    realm?: string;
+    password?: string;
+    firstName?: string;
+    lastName?: string;
+    groups?: string[];
+    enabled?: boolean;
+  }) {
+    const payload: {
+      credentials?: { type: string; value: string; temporary: boolean }[];
+      firstName?: string;
+      lastName?: string;
+      groups?: string[];
+      enabled?: boolean;
+    } = {};
+    if (password) {
+      payload.credentials = [
+        {
+          type: 'password',
+          // must follow the password policy on keycloak
+          value: password || uid(12) + '@123',
```

Comment:
So it possible that Okta send us no password?

Reply:
yep, in certain routes they dont

Reply:
In what routes? This route is only about to provision users, isn't?

Reply:
in the patch route, they dont send